plaso/containers/tasks.py

```diff
@@ -35,6 +35,7 @@ class Task(interface.AttributeContainer):
     start_time (int): time that the task was started. Contains the number
         of micro seconds since January 1, 1970, 00:00:00 UTC.
     storage_file_size (int): size of the storage file in bytes.
+    storage_format (str): the format the task was stored in.
```

Comment:
was stored in? or should be stored in?

is this used for informational purposes or for configuration purposes?